lib/elixir/lib/module.ex

```diff
@@ -1601,6 +1601,7 @@ defmodule Module do
       end
 
     :ets.insert(table, {key, typespecs, true, nil})
+    nil
```

Comment:
Another way would be (maybe better) to return it dependant from a successful insert like:

`:ets.insert(..) && nil`